wpilib/wpilib/shuffleboard/container.py

```diff
@@ -42,83 +48,76 @@ def getLayout(self, type: str, title: str):
 
         return self.layouts[title]
 
-    def add(self, *args, **kwargs) -> "ShuffleboardWidget":
+    @overload
```

Comment:
... heh

Reply:
Most appropriate way to document how one would get the different return types, I guess (even though the overload signatures don't show up in the docs). :man_shrugging: